netplan_cli/cli/ovs.py

```diff
@@ -125,6 +125,9 @@ def apply_ovs_cleanup(config_manager, ovs_old, ovs_current):  # pragma: nocover
     Also filter for individual settings tagged netplan/<column>[/<key]=value
     in external-ids and clear them if they have been set by netplan.
     """
+    if not systemctl_is_installed(OPENVSWITCH_OVSDB_SERVER_UNIT):
+        logging.debug("Cannot apply OVS cleanup: %s is 'not-found'.", OPENVSWITCH_OVSDB_SERVER_UNIT)
+        return False
```

Comment:
Hmm this method doesn't really return a value. This is the only explicit return point in the code. We probably should throw an exception here and handle it as we handle `OvsDbServerNotRunning`.

Reply:
Fair point! I've updated it accordingly, catching the error in `apply.py`, then logging a debug message that is usually muted.

@daniloegea What do you think now?